client/platform/web-girder/views/Export.vue

```diff
@@ -34,15 +34,18 @@ export default defineComponent({
   setup(props) {
     const menuOpen = ref(false);
     const excludeFiltered = ref(false);
+    const excludeUncheckedTypes = ref(false);
     const exportUrls = ref(null as null | ExportUrlsResponse);
     const savePrompt = ref(false);
     const currentSaveUrl = ref('');
     let save = () => Promise.resolve();
     let pendingSaveCount = ref(0);
+    let checkedTypes = ref([] as readonly string[]);
 
     if (props.blockOnUnsaved) {
       save = useHandler().save;
       pendingSaveCount = usePendingSaveCount();
+      checkedTypes = useCheckedTypes();
```

Comment:
I don't know if this should be in the `blockOnUnsaved` prop and maybe it should be outside of it where the variable is initialized?  I know `blockOnUnsaved` set outside of here in the `ViewLoader.vue` but the default properly is false.  `checkedTypes` once bound in the beginning should auto update anytime the checked types are changed.  I could be mistaken on this and it's handling an edge case I don't know about.

Reply:
`blockOnUnsaved` is a proxy indicator for "this export dialog was mounted on the viewer instead of the data browser".

If this were outside this block, it would cause a runtime exception because the Provides api was not injected.

There's a conditional on menuOpen below to prevent requests to the server unless the menu is actually open.

No changes needed, IMO.

Reply:
thanks, that was the case I was missing.